ocaml/xapi/sm_exec.ml

```diff
@@ -320,7 +320,7 @@ let with_session sr f =
       finally (fun () -> f session_id) (fun () -> destroy_session session_id)
   )
 
-let exec_xmlrpc ?context:_ ?(needs_session = true) (driver : string)
+let exec_xmlrpc ?dbg ?context:_ ?(needs_session = true) (driver : string)
```

Comment:
I think there are no callers of this function without `~dbg` anymore after both commits? Then we should make this a non-optional argument.

Reply:
Done in here https://github.com/xapi-project/xen-api/pull/5312/commits/4fced77c8c38335d315d72242048172f3d7578d9 and here https://github.com/xapi-project/xen-api/pull/5312/commits/366c55dff31b8be28aef9dc632ba86743944d277.